### ISSUE TITLE
[BUG] Blank replies on API.

### ISSUE THREAD

**Issue by NigaKniga** [SillyTavern/SillyTavern#1880]
### Environment

Self-Hosted (Bare Metal)

### System

Any browser

### Version

1.11.5

### Describe the problem

Extremely frequent blank replies from Claude (any version, even 1.3 when it was still available). Sometimes happens with GPT4 too.
This is a silly tavern issue, not an API one, because it never happens on Silly version 1.9, when you switched to a new branch, this issue started appearing and I wonder if you are even aware of it.
Console outputs - "Streaming Request finished" and I get a blank message. Silly also sometimes cuts the message off in the middle (Happens more with GPT4).

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You've checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
Disable streaming and see what exactly is being cut in the console window. It's extremely likely to be an impersonated reply that starts `{{char}}:`, means that it is probably an issue with how you prompt.
Add more information like exact prompts and other settings. You can enable "Allow {{char}} in responses" in user settings and try again.

**Comment by NigaKniga** [comment]
I did what you suggested and it changed nothing. It's not my prompts, I have tried 2 separate ones. I consistently get no blanks on old silly tavern version and 6\8 blanks on a new one, no matter the prompt.
There seem to be something wrong with the way silly receives output from proxy. Cutting the message off prematurely also hints on some issue with that.

I hope this narrows down the possible cause of the problem. Settings does not seem to affect this, because vanilla silly behaves in the same way. No issues on old silly, blanks on any of the new branch versions.
